app/src/common/shared/org/mozilla/vrbrowser/SessionStore.java

```diff
@@ -77,10 +73,58 @@ public static SessionStore get() {
         GeckoSession mSession;
     }
 
+    private static class GeckoError {
+
+        public static final String ERROR_URL = "resource://android/assets/html/error/netError.xhtml";
+
+        public static final String NET_ERROR_TYPE = "about:neterror";
```

Comment:
https://revoked.badssl.com should be blocked as a `neterror`, but it isn't:

![revoked badssl com](https://user-images.githubusercontent.com/203725/43291222-887332a8-90e5-11e8-8713-a97c1045714a.png)
![image](https://user-images.githubusercontent.com/203725/43291231-8ce9c1a8-90e5-11e8-8036-1202a90a6851.png)

Reply:
This seems to be a Gecko issue, we are not getting an error for this one.